filebeat/docs/modules/misp.asciidoc

```diff
@@ -0,0 +1,30 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-misp]]
+[role="xpack"]
+
+:modulename: misp
+:has-dashboards: false
+
+== MISP module
+
+beta[]
+
+This is a filebeat module for reading threat intel information from the MISP platform (https://www.circl.lu/doc/misp/). It uses the httpjson input to access the MISP REST API interface.
+
+The configuration in the config.yml file uses the following format:
+    var.api_key: specifies the API key to access MISP.
```

Comment:
I think there's a bit of a formatting issue with the config example.

Reply:
Do you mean the indent before var.api.key? It is done on purpose. Maybe not the best way to do it?

Reply:
Yeah, I mean the indentation. I don’t think this is what you we’re going for.

![78AC7DBC-CEED-4090-A7D6-222523E4EA88](https://user-images.githubusercontent.com/4565752/66820818-5ddfb280-ef0f-11e9-8dab-22daaa072412.jpeg)